gcloud/orb.yaml

```diff
@@ -397,6 +397,11 @@ jobs:
         description: >
           Extra flags to pass to docker build.
         type: string
+      custom-checkout:
+        default: ''
+        description: >
+          Pull repositories other than tooling for building.
```

Comment:
Please add a comment re default behaviour, eg. something like "if unset, uses the current repository". Probably also worth including other caveats, eg. "...pulls the default branch from..." and such